spec/mailers/user_mailer_spec.rb

```diff
@@ -762,6 +764,58 @@ def expect_email_body_to_have_help_and_contact_links
         end
       end
 
+      context 'Need to change location section' do
+        context 'when Enhanced IPP is not enabled' do
+          let(:is_enhanced_ipp) { false }
+          let(:mail) do
+            UserMailer.with(user: user, email_address: email_address).in_person_ready_to_verify(
+              enrollment: enhanced_ipp_enrollment,
+              is_enhanced_ipp: is_enhanced_ipp,
+            )
+          end
+          it 'renders the change location heading' do
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.body.location.change_location_heading'),
+            )
+          end
+
+          it 'renders the change location info' do
+            expect(mail.html_part.body).to have_content(
+              t(
+                'in_person_proofing.body.location.change_location_info_html',
+                find_other_locations_link_html:
+                  t('in_person_proofing.body.location.change_location_find_other_locations'),
+              ),
+            )
+          end
+        end
+        context 'when Enhanced IPP is enabled' do
+          let(:is_enhanced_ipp) { true }
+          let(:mail) do
+            UserMailer.with(user: user, email_address: email_address).in_person_ready_to_verify(
+              enrollment: enhanced_ipp_enrollment,
+              is_enhanced_ipp: is_enhanced_ipp,
+            )
+          end
+
+          it 'does not render the change location heading' do
+            expect(mail.html_part.body).not_to have_content(
+              t('in_person_proofing.body.location.change_location_heading'),
+            )
+          end
+
+          it 'does not render the change location info' do
+            expect(mail.html_part.body).not_to have_content(
+              t(
+                'in_person_proofing.body.location.change_location_info_html',
+                find_other_locations_link_html:
+                  t('in_person_proofing.body.location.change_location_find_other_locations'),
+              ),
+            )
+          end
+        end
+      end
+
```

Comment:
Non-blocking nit request: While you're in this file, could you please change line 819 to specify that it's Informed Delivery IPP (ID-IPP)?